src/ancillary/__tests__/anchor-request-params-parser.test.ts

```diff
@@ -125,6 +136,16 @@ describe('AnchoRequestParamsParser', () => {
 
     const params: RequestAnchorParams = (validation as Right<RequestAnchorParams>).right
     expect(params.cacaoDomain).toEqual(FAKE_CACAO)
+    expect((params as RequestAnchorParamsV2).genesisFields.model?.toString() ?? '').toEqual('')
```

Comment:
```suggestion
    expect((params as RequestAnchorParamsV2).genesisFields.model).toBeUndefined()
```

Reply:
well - the way i wrote the code it will be '', not undefined.  I did that because we are logging it and it needs to be a string.

Reply:
the reason i put this extra cruft in the test was to exercise the similar code in constructing the log data.  but, I agree, it is not really testing the code rather than reproducing it!  I will change the test, but i did  need it this way to make sure it won't break since i didn't have a good test to actually exercise the whole request